src/_guides/libraries/library-tour.md

```diff
@@ -833,8 +833,11 @@ y2k = DateTime.utc(2000); // 1/1/2000, UTC
 // Specify a date and time in ms since the Unix epoch.
 y2k = DateTime.fromMillisecondsSinceEpoch(946684800000, isUtc: true);
 
-// Parse an ISO 8601 date.
+// Parse an ISO 8601 date in the UTC time zone.
 y2k = DateTime.parse('2000-01-01T00:00:00Z');
+
+// Create a new DateTime from an existing one, adjusting just some properties:
+var sameTimeLastYear = now.copyWith(year: now.year - 1);
 ```
 
```

Comment:
Based on [this comment](https://github.com/dart-lang/sdk/issues/24644#issuecomment-1260958607), should we add a warning here (similar to the one on lines 879-883)?

```suggestion
{{site.alert.warning}}
  `DateTime` operations might give unexpected results related to Daylight Savings Time and other non-standard time adjustments.  
{{site.alert.end}}
```

@lrhn @mit-mit @spydon